### ISSUE TITLE
Kanban Board does not load

### ISSUE THREAD

**Issue by jsc57x** [joplin/plugin-kanban#27]
I tried to create a test kanban board but the window only shows "Loading..."
![image](https://user-images.githubusercontent.com/11822263/157324099-67a2b6c4-d5cf-4773-9f88-40a7b8c6ee2e.png)
Did I miss something?
Im using Joplin 2.7.13 and Kanban 1.0.7

**Comment by mapclyps** [comment]
I had the same issue after an update of Joplin and the Kanban plugin. It was resolved for me with removing and installing the plugin again.

**Comment by mablin7** [comment]
Hi @jsc57x !

Is your situation similar to @mapclyps 's? Could your kanban logs? Here's how to get it:
 1. I recommend you reproduce the issue in an empty notebook as the logs can contain the titles of the notes on the board, which may be personal.
 1. Inside Joplin, click `Help` > `Open profile directory`
 2. You should see a `kanban-logs.txt` file. Please upload it here.

**Comment by jsc57x** [comment]
Yeah it works now. Thanks alot for the tip. I only deactivated and activated it. Logs are attached
[kanban-logs.txt](https://github.com/joplin/plugin-kanban/files/8310839/kanban-logs.txt)

**Comment by shinglyu** [comment]
I had similar issue, after digging into the log I found out it's an YAML parsing error:
```yaml
columns:
  - name: Priority-3 (Backlog)
    backlog: true
```
needs to be quoted:
```yaml
columns:
  - name: "Priority-3 (Backlog)"
    backlog: true
```

Maybe we can show this kind of error message in the Kanban panel?

**Comment by PeterParkerWEB3** [comment]
I still get the YAML parse error - even with quotes...? Logs are in place. Using Joplin on linux Version 2.8.8 & kanban 1.0.7 
Any ideas? Would love the feature :)

![Screenshot from 2022-10-07 17-59-37](https://user-images.githubusercontent.com/115272942/194597464-58ddbc08-fb28-42a2-8b7d-8ef1a48cf817.png)


![Screenshot from 2022-10-11 21-31-29](https://user-images.githubusercontent.com/115272942/195182021-095f9358-860c-4940-8585-dc8d39cbc308.png)

**Comment by frederikstroem** [comment]
> I still get the YAML parse error - even with quotes...? Logs are in place. Using Joplin on linux Version 2.8.8 & kanban 1.0.7 Any ideas? Would love the feature :)
> 
> ![Screenshot from 2022-10-07 17-59-37](https://user-images.githubusercontent.com/115272942/194597464-58ddbc08-fb28-42a2-8b7d-8ef1a48cf817.png)
> 
> ![Screenshot from 2022-10-11 21-31-29](https://user-images.githubusercontent.com/115272942/195182021-095f9358-860c-4940-8585-dc8d39cbc308.png)

@PeterParkerWEB3 I think you need to add proper indentation. I just used the space-bar until everything was aligned.

For instance, my setup is like this:
![correct](https://user-images.githubusercontent.com/17912119/219312706-bcd861f6-083b-4d3e-adf2-1a0ca5da8847.png)

The above setup works, however, if the indentation is wrong, the plugin will throw a *"YAML Parse error"*. This fails for example (notice the selected text):
![wrong](https://user-images.githubusercontent.com/17912119/219313688-d9fff261-4881-4db4-bfbf-c65b70c1784c.png)
